utils/groovy/src/main/java/org/apache/brooklyn/util/groovy/GroovyJavaMethods.java

```diff
@@ -75,6 +76,20 @@
         }
     }
 
+    /**
+     * Alternative implementation of {@link ScriptBytecodeAdapter#isCase(Object, Object)}<br>
+     * Stripped down to work only for caseExpression of type <code>java.lang.Class</code>.<br>
```

Comment:
If we're OK to do that then do we even need this method? Couldn't we just use instanceof/isAssignableFrom as appropriate?
```java
if ( job instanceof Callable) {
```
etc?